socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+    <div class="grid-item-author-bar mt-1">
+        <div @click.stop.prevent="profileBoxTrigger" class="profile-box-trigg">
```

Comment:
True, nasty conflicts.. We could separate `streams.js` out from the compiled `project.js` and not load that here but hmm... Maybe call this version `profilebox-trigger` and leave the jQuery one alone?